src/net_processing.cpp

```diff
@@ -2334,6 +2342,8 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         return;
     }
 
+    PeerRef peer = GetPeerRef(pfrom.GetId());
+    assert(peer);
```

Comment:
I think it's probably safer to just return if you can't find the `Peer` object here (as all other instances of `GetPeerRef()` except the `FinalizeNode()` call do) rather than assert.

Reply:
addressed.